src/main/java/redis/clients/jedis/resps/KeyedZSetElement.java

```diff
@@ -0,0 +1,43 @@
+package redis.clients.jedis.resps;
+
+import redis.clients.jedis.Tuple;
+import redis.clients.jedis.util.SafeEncoder;
+
+public class KeyedZSetElement extends Tuple {
```

Comment:
@sazzad16 I think some JavaDoc that will describe this class is needed

Reply:
@gkorland JavaDoc added.